pkg/imagedistributor/mount.go

```diff
@@ -19,22 +19,26 @@ import (
 	"net"
 	"os"
 	"path/filepath"
+	"strings"
 
-	"github.com/sealerio/sealer/common"
 	imagecommon "github.com/sealerio/sealer/pkg/define/options"
 	"github.com/sealerio/sealer/pkg/imageengine"
 	v1 "github.com/sealerio/sealer/types/api/v1"
 	osi "github.com/sealerio/sealer/utils/os"
 	"github.com/sealerio/sealer/utils/os/fs"
 )
 
+const (
+	mountBaseDir = "/var/lib/sealer/data/mount"
```

Comment:
move it to. common/common.go file?

Reply:
mountBaseDir is internal used , so i dont move it to pubic common.